h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -412,6 +416,7 @@ public BinomialModelPrediction predictBinomial(RowData data, double offset) thro
     double[] preds = preamble(ModelCategory.Binomial, data, offset);
 
     BinomialModelPrediction p = new BinomialModelPrediction();
+    p.leafNodeAssignments = leafNodeAssignment(data);  // assign leaf node assignment if desired
```

Comment:
This should be optional, it will slow down the prediction.

I am thinking of having a separate API for this feature. We should not include it in the default response.

Reply:
Agreed.  It will only be executed if called separately and with the correct mojo-version.

Reply:
I added a parameter to EasyPredictModelWrapper.java, boolean getLeafAssignment which is default to be false.  If a user want to get the leaf node assignment, they will need to call enableLeafAssignment() to set it to true.